datafiles/README.md

```
@@ -0,0 +1,37 @@
+
+## Where are the save files stored?
+
+`C:\Users\<UserName>\AppData\Local\ChapterMaster\`.
+
+## Where are the error logs stored?
+
+`C:\Users\<UserName>\AppData\Local\ChapterMaster\Logs`.
+
+## Where are the message logs stored?
+
+`C:\Users\<UserName>\AppData\Local\ChapterMaster\Logs\last_messages.log`.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)